lighthouse-cli/test/smokehouse/perf/run-tests.sh

```diff
@@ -6,8 +6,13 @@ sleep 0.5s
 
 config="lighthouse-core/config/perf.json"
 expectations="lighthouse-cli/test/smokehouse/perf/expectations.js"
+save_assets=""
 
-npm run -s smokehouse -- --config-path=$config --expectations-path=$expectations
+if [[ "$CI" = true ]]; then
+  save_assets="--save-assets-path=perf.json"
```

Comment:
can you add a comment here and in `.travis.yml` why this is being done. e.g. "Save assets so that traces of test failures can be examined" or something